src/flamenco/runtime/fd_runtime.c

```diff
@@ -2830,6 +2830,36 @@ fd_runtime_collect_rent_for_slot( fd_exec_slot_ctx_t * slot_ctx, ulong off, ulon
   }
 }
 
+/* Yes, this is a real function that exists in Solana. Yes, I am ashamed I have had to replicate it. */
+// https://github.com/firedancer-io/solana/blob/d8292b427adf8367d87068a3a88f6fd3ed8916a5/runtime/src/bank.rs#L5618
+static ulong
+fd_runtime_slot_count_in_two_day( ulong ticks_per_slot ) {
+  return 2UL * FD_SYSVAR_CLOCK_DEFAULT_TICKS_PER_SECOND * 86400UL /* seconds per day */ / ticks_per_slot;
+}
+
+static ulong
+fd_runtime_num_rent_partitions( fd_exec_slot_ctx_t const * slot_ctx, ulong slot ) {
```

Comment:
If possible, could we add a permalink for this, if it corresponds to a chunk of Agave code? It would be great to do this whenever we change runtime code.

Reply:
done